### PR TITLE
move slava77 from reco L2 to tracking-pog, keep the PR test role

### PR DESCRIPTION
**Pull request by slava77**
@smuzaffar 
do I also need to update the cmssw_l2/l2.json or is it automated?

### CODE REVIEW

**Comment by smuzaffar** [comment]
> do I also need to update the cmssw_l2/l2.json or is it automated?

no that will be updated automatically

File: categories.py [hunk 1]
```diff
@@ -131,7 +130,7 @@
   "kshcheli":         ["proton-pog"],
   "kandrosov":        ["tau-pog"],
   "alebihan":         ["tau-pog"],
-  "mtosi":            ["tracking-pog"],
```
Comment (smuzaffar) [inline categories.py:134]: oops, did not notice this change. @slava77 was this an intentional update?

Comment (slava77) [reply categories.py:134]: yes, it was intentional.
@mtosi @mmusich @vmariani 
may want to clarify if there were some longer term arrangements

Comment (mmusich) [reply categories.py:134]: there have been no such arrangements.

Comment (smuzaffar) [reply categories.py:134]: so all is good then

**Pull request merged**